src/screens/TransactionConfirmationScreen.js

```diff
@@ -416,6 +417,10 @@ export default function TransactionConfirmationScreen() {
 
   const closeScreen = useCallback(
     canceled => {
+      // we need to close the hw navigator too
+      if (accountInfo.isHardwareWallet) {
+        goBack();
+      }
```

Comment:
Oh OK, so we need to call it twice. Hate to say it, but that's pry more good spots for comments lol.

Reply:
yeah good callout